src/test/java/core/basesyntax/handlers/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.handlers;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static OperationHandler operationHandler;
+
+    @Before
+    public void setUp() {
+        operationHandler = new BalanceOperationHandler(new StorageDaoImpl());
+    }
+
+    @Test
+    public void balance_correct_Ok() {
```

Comment:
```suggestion
    public void doOperation_balance_Ok() {
```

There are a lot of ways to name your test methods. The main point is that they should have informative names and be consistent along with other developers in your team. For this task use such convention: `<methodUnderTest>_<state>_<expectedBehavior>`; For example, if we are testing the method register with a `null` user's age the test method name should be `register_nullAge_notOk`. `notOk` is because the test expects the register method to throw an exception.

fix all tests names